### DISCUSSION TITLE
Change log/webhook-display names of previous and updated choicefield instead of uuid

### DISCUSSION THREAD

**Discussion by naga293tach** [nautobot/nautobot#1930] [General]
can you please modify the changelog to display names of the previous and the updated choice field in th ui/webhook(eg: status, device role ,etc.) instead of uuid of the objects so that it will be user friendly.

**Comment by bryanculver** [comment]
Thanks for opening the discussion @naga293tach! We already have a feature request open for this: https://github.com/nautobot/nautobot/issues/1830 which may also be a duplicate of/be implemented by: https://github.com/nautobot/nautobot/issues/1530
